packages/bpk-component-modal/package.json

```diff
@@ -16,7 +16,7 @@
     "bpk-component-close-button": "^2.0.35",
     "bpk-component-icon": "^5.0.35",
     "bpk-component-link": "^2.0.34",
-    "bpk-component-navigation-bar": "^2.1.31",
+    "bpk-component-navigation-bar": "^2.1.32",
```

Comment:
You won't need to update this as this will be done during the release process 😄

Reply:
Reverted!

Reply:
And re-reverted